doc/quick-start/qs-guide.rst

```diff
@@ -7,82 +7,119 @@ This guide will help you to set up thing's development environment, then build a
 
-         $ knot board dk
+   .. code-block:: bash
+
+      cd ~/zephyr-knot-sdk/apps/hello-dongle
+
+#. Run environment image.
+
+   .. code-block:: bash
 
-   - If using the Dongle:
+      docker run -ti -v $(pwd)/:/workdir cesarbr/knot-zephyr-sdk:latest
 
-      - Connect the board to a USB port.
-      - Press the reset button. The red led will blink.
-      - Set the Dongle as the default target board.
+#. From the container, build the project for the target board.
+
+   - If using the `KNoT DK <https://docs.zephyrproject.org/latest/boards/arm/nrf52840_pca10056/doc/index.html>`_:
 
       .. code-block:: bash
 
-         $ knot board dongle
+         [user@container] $ knot make --board dk
 
-Build KNoT Hello App
-''''''''''''''''''''
+   - If using the `KNoT Dongle <https://docs.zephyrproject.org/latest/boards/arm/nrf52840_pca10059/doc/index.html>`_:
 
-- Go to the KNoT Hello directory.
+      .. code-block:: bash
 
-   .. code-block:: bash
+         [user@container] $ knot make --board dongle
 
-      $ cd $KNOT_BASE/apps/hello
 
-.. note:: If you are using the Dongle go to hello-dongle directory.
+Flashing board
+''''''''''''''
 
-- Build and flash apps.
+1. From your project folder, export the generated files to a ``output`` folder.
 
    .. code-block:: bash
 
-      $ knot make --mcuboot
+      [user@container] $ knot export output/
+
+#. Install `nRF Connect <https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF-Connect-for-desktop/Download>`_.
+
+#. Open *nRF Connect* and add the *Programmer App*.
+
+   .. image:: ../../_static/nrfconnect_add_programmer.png
+      :scale: 70 %
+      :alt: nRF Connect: Add Programmer
+      :align: center
+
+#. Launch the *Programmer App*.
+
+   .. image:: ../../_static/nrfconnect_launch_programmer.png
+      :scale: 70 %
+      :alt: nRF Connect: Launch Programmer
+      :align: center
+
+#. Connect the device to a USB port.
+
+   If using the `KNoT Dongle <https://docs.zephyrproject.org/latest/boards/arm/nrf52840_pca10059/doc/index.html>`_, press the *RESET* button to get into DFU mode.
+   The red LED will start to blink.
 
-.. note:: The option 'mcuboot' flashes the compiled program and the mcuboot bootloader at the end of building.
+#. Select the target device.
 
-Monitor the output
-''''''''''''''''''
+   .. image:: ../../_static/nrfconnect_select_device.png
+      :scale: 70 %
+      :alt: nRF Connect: Select device
+      :align: center
 
-You can use minicom or any other serial port reader to monitor the app output.
+#. Define the HEX file to be flashed.
 
-- Install minicom
+   Click **Add HEX file** and select the ``boot_sgn_apps.hex`` file that was exported to the ``output/`` folder.
 
-.. tip:: You can install minicom on debian based systems by using: ``$ sudo apt-get install minicom``
+   .. image:: ../../_static/nrfconnect_add_hex.png
+      :scale: 70 %
+      :alt: nRF Connect: Add HEX file
+      :align: center
```

Comment:
Is there any way to put a space (break line) here? The image is connected with the text below.

Reply:
An approach could be the use of the figure directive and maybe add a subtitle on it.

Reply:
I followed the figure approach. Thank you, @alla3 .